rmf_task/src/rmf_task/requests/Clean.cpp

```diff
@@ -82,10 +82,10 @@ rmf_task::DescriptionPtr CleanDescription::make(
       clean->_pimpl->motion_sink->compute_change_in_charge(cleaning_path);
     const double dSOC_ambient =
       clean->_pimpl->ambient_sink->compute_change_in_charge(
-        rmf_traffic::time::to_seconds(clean->_pimpl->invariant_duration));
```

Comment:
I've tried to find the right settings to achieve nested indents with `uncrustify` but was unsuccessful. Perhaps you could give it a shot in the future.